exercises/printUserInfo.js

```diff
@@ -7,11 +7,13 @@ let process = require('process');
 
 let user = {};
 
-user['firstName'] = '<Change This>';
-user['lastName'] = '<Change This>';
-user['currentLocation'] = '<Change This>';
-user['favoriteSongs'] = ['One Song', 'Another Great Song', 'Last Great Song'];
-user['age'] = 108;
+user['firstName'] = 'Stephen';
+user['lastName'] = 'Strange';
+user['currentLocation'] = 'New York';
+user['favoriteSongs'] = ['Smote and Mirrors', 'The Eyes Have It', 'Hippocratic Hypocrite'];
+user['age'] = 40;
+user['favoriteFood'] = ['Onigiri'];
```

Comment:
You *probably* don't want this to be a single-element array containing a `string`, but rather a just a `string`. No need to do `toString()` later.